admin/win/msi/Nextcloud.wxs

```diff
@@ -45,7 +45,7 @@
         https://www.joyofsetup.com/2010/01/16/major-upgrades-now-easier-than-ever/
     -->
     <MajorUpgrade Schedule="afterInstallInitialize" AllowDowngrades="yes" />
-    <Property Id="REINSTALLMODE" Value="amus" />
+    <Property Id="REINSTALLMODE" Value="omus" />
```

Comment:
hmm, I wonder if this shouldn't be `dmus` instead

if I understand the [REINSTALLMODE flags](https://learn.microsoft.com/en-us/windows/win32/msi/reinstallmode) correctly:

- `o` replaces files if they are an older version (might make downgrades difficult?)
- `d` replaces files if the version is different
- `e` replaces files if the version is the same as the current one or older
- `a` replaces all files regardless